src/twisted/web/client.py

```diff
@@ -43,6 +47,17 @@
     IResponse,
 )
 
+# Allow for type-checking of our faked-out types.
```

Comment:
Maybe something like this.

```suggestion
# For the purpose of type-checking we want our faked-out types to be identical to the types they are replacing.
# For the purpose of the impementation, we want to start
# with a blank slate so that we don't accidentally use
# any of the real implementation.
```